docs/catalog/ai/serverless-model-development.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Adopt serverless architecture for ML workload process
+
+## Description
+
+Building an ML model requires a series of steps, like building
+a data pipeline for data capture, data cleansing, feature generation, and running multiple training iterations and experiments to get the desired accuracy. All of these steps take significant computing resources that need to be optimized for efficient utilization.
+
+
+## Solution
+Adopt a serverless architecture for maximum resource utilization for your entire AI/ML model development process like data pipeline, training and experiments. Serverless ensures the resources are launched only when required. 
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+For the SCI equation, adopting serverless architecture for AI/ML workload process would impact the following:
+
+- 'E':  Having a serverless architecture for the AI/ML development process provdies efficient resource utilization and reduces the energy consumption of the compute resources and consequently, the E number should decrease.
```

Comment:
There is also an `M` to account for here, especially from the customer perspective. Only using hardware for the workload volume will reduce the overall embodied carbon associated to the AI/ML process.

Reply:
Added M. Thanks